stable/coredns-updater/templates/coredns-updater-deployment.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "coredns-updater.fullname" . }}
+  namespace: iguazio-platform
+  labels:
+    {{- include "coredns-updater.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "coredns-updater.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "coredns-updater.selectorLabels" . | nindent 8 }}
+      name: coredns-update
+      namespace: iguazio-platform
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "coredns-updater.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args: ["-n", "app" ]
+        livenessProbe:
+          exec:
+            command:
+              - python
+              - health_check.py
+            initialDelaySeconds: 30
+            periodSeconds: 600
```

Comment:
id avoid `exec`ing commands for liveness probe. 

<img width="711" alt="Screen Shot 2022-03-15 at 9 12 55" src="https://user-images.githubusercontent.com/6547459/158325406-c91c8e87-ef03-487f-98aa-78d7129b0e9a.png">

thought it has been fixed on k8s > 1.20, we might use your helm/application on earlier versions of k8s.
thus, i suggest having a small http server running within your app, serving response for healthiness.
talk to me offline if you need help with that

Reply:
Fixed